recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend

```diff
@@ -7,14 +7,12 @@ SRC_URI_append = " \
 
 # put it out of the source tree, so it can be reused after cleanup
 ANDROID_OUT_DIR_COMMON_BASE = "${SSTATE_DIR}/../${ANDROID_PRODUCT}-${SOC_FAMILY}"
-ANDROID_PRODUCT_OUT = "${ANDROID_OUT_DIR_COMMON_BASE}/target/product/${ANDROID_PRODUCT}"
+ANDROID_PRODUCT_OUT="${WORKDIR}/repo/out/target/product/xenvm"
```

Comment:
why do we need this variable ?
ANDROID_PRODUCT_OUT="${WORKDIR}/repo/out/target/product/xenvm"